remo/base/views.py

```diff
@@ -42,10 +44,20 @@ def login_failure(self, msg=''):
 def main(request):
     """Main page of the website."""
     featured_rep = utils.latest_object_or_none(FeaturedRep)
-    feed = feedparser.parse(settings.PLANET_URL)
-    planet_entries = feed.entries[:3]
+
+    resp = cache.get('planet')
+    if not resp:
```

Comment:
Instead of having both `if not resp` and `if resp`, this could be written:

```python
planet_feed = cache.get('planet')

if not planet_feed:
    try:
         planet_feed = reqeuests.get(settings.PLANET_URL, timeout=settings.PLANET_MAX_TIMEOUT).text
    except (first_exception, second_exception):
         planet_feed = ''
    else:
         cache_set('planet, planet_feed, 60 * 60 * 8)
results = feedparser.parse(unicode(planet_feed)).entries[:3]
```